Modules/Scripted/DICOMLib/DICOMExportScene.py

```diff
@@ -148,7 +148,9 @@ def createDICOMFileForScene(self):
         # cmd = "img2dcm -k 'SeriesDescription=Slicer Data Bundle' -df %s/template.dcm %s %s" % (saveDirectoryPath, imageFile, self.sdbFile)
         seriesDescription = 'Slicer Data Bundle' if self.seriesDescription is None else str(self.seriesDescription)
         args = [
-            '-k', f'SeriesDescription={seriesDescription}']
+            '-k', f'SeriesDescription={seriesDescription}',
+            '--no-latin1']  # With this option the UTF8 encoding of the reference file is preserved, otherwise the character set would be forced to Latin1
+        #TODO: It could be safer to pass these fields through the reference file instead of using command-line arguments (from character set point of view)
```

Comment:
A linting error https://github.com/Slicer/Slicer/actions/runs/5154914330/jobs/9283929557?pr=6963
```suggestion
        # TODO: It could be safer to pass these fields through the reference file instead of using command-line arguments (from character set point of view)
```

Reply:
Also if this is actually future TODO work, maybe write up a GitHub issue to track it instead? In line TODOs seem to get lost in a big project

Reply:
@lassoan suggested adding a comment about this https://github.com/Slicer/Slicer/pull/6963#issuecomment-1572286973

All I want is get this into Slicer. Thanks.